connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorSourceTaskTest.java

```diff
@@ -214,6 +225,82 @@ public void testPoll() {
         }
     }
 
+    @Test
+    public void testSeekBehaviorDuringStart() {
+        // Setting up mock behavior.
+        @SuppressWarnings("unchecked")
+        KafkaConsumer<byte[], byte[]> mockConsumer = mock(KafkaConsumer.class);
+
+        @SuppressWarnings("unchecked")
+        KafkaProducer<byte[], byte[]> mockProducer = mock(KafkaProducer.class);
+
+        String sourceClusterName = "sourceCluster";
+        MirrorSourceMetrics mockMetrics = mock(MirrorSourceMetrics.class);
+
+        SourceTaskContext mockSourceTaskContext = mock(SourceTaskContext.class);
+        OffsetStorageReader mockOffsetStorageReader = mock(OffsetStorageReader.class);
+        when(mockSourceTaskContext.offsetStorageReader()).thenReturn(mockOffsetStorageReader);
+
+        MockedStatic<MirrorUtils> mockMirrorUtils = mockStatic(MirrorUtils.class, new CallsRealMethods());
```

Comment:
I don't think we'll need this if we adopt the isolated `initializeConsumer` method, but just in case, we can tweak this to use official API from Mockito instead of an internal class:
```suggestion
        MockedStatic<MirrorUtils> mockMirrorUtils = mockStatic(MirrorUtils.class, CALLS_REAL_METHODS);
```

(requires a static import for `org.mockito.Mockito.CALLS_REAL_METHODS`)